source/patterns/@aws-solutions-constructs/aws-apigatewayv2websocket-sqs/test/apigatewayv2websocket-sqs.test.ts

```diff
@@ -0,0 +1,339 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import * as cdk from "aws-cdk-lib";
+import * as apigwv2 from "aws-cdk-lib/aws-apigatewayv2";
+import * as lambda from "aws-cdk-lib/aws-lambda";
+
+import { Capture, Match, Template } from "aws-cdk-lib/assertions";
+import { WebSocketIamAuthorizer } from "aws-cdk-lib/aws-apigatewayv2-authorizers";
+import { WebSocketLambdaIntegration } from "aws-cdk-lib/aws-apigatewayv2-integrations";
+import { ApiGatewayV2WebSocketToSqs } from "../lib";
+import { COMMERCIAL_REGION_LAMBDA_NODE_RUNTIME } from "@aws-solutions-constructs/core";
+
```

Comment:
What about adding another test that verifies the default authorizer is created by default? apologies if its already in here.

Reply:
Removed authorizer. Hence resolving this comment.